lib/action_controller/serialization.rb

```diff
@@ -25,7 +25,8 @@ def get_serializer(resource, options = {})
           "Please pass 'adapter: false' or see ActiveSupport::SerializableResource.new"
         options[:adapter] = false
       end
-      serializable_resource = ActiveModel::SerializableResource.new(resource, options)
+      serializable_options = options.merge(controller: self)
```

Comment:
If you make it easy to use the controller directly, people will, and the scope will creep and and new ams will gain responsibilities it shouldn't gave and cause pain and bugs to users and maintainers 

Failure of Interface segregation, not keepig it narrow

B mobile phone

> On Sep 24, 2015, at 9:40 PM, Lucas Hosseini <notifications@github.com> wrote:
> 
> In lib/action_controller/serialization.rb:
> 
> > @@ -25,7 +25,8 @@ def get_serializer(resource, options = {})
> >            "Please pass 'adapter: false' or see ActiveSupport::SerializableResource.new"
> >          options[:adapter] = false
> >        end
> > -      serializable_resource = ActiveModel::SerializableResource.new(resource, options)
> > +      serializable_options = options.merge(controller: self)
> Currently we just need the class name but there's virtually 0 overhead to getting the instance, so why not?
> 
> —
> Reply to this email directly or view it on GitHub.
>